public/room.html

```diff
@@ -0,0 +1,128 @@
+          <a class="navbar-brand" href="index.html">Powell County Schools</a>
+        </div>
+        <div id="navbar" class="navbar-collapse collapse">
+          <ul class="nav navbar-nav">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Facilities <span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li><a href="building.html">Building</a></li>
+                <li class="active"><a href="room.html">Room</a></li>
+              </ul>
+            </li>
+            <li><a href="#">Inventory</a></li>
+            <li><a href="#">Check-In</a></li>
+            <li><a href="#">Reports</a></li>
+          </ul>
+          <ul class="nav navbar-nav navbar-right">
+            <li><a href="login.html">Log out</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-sm-3 col-md-2 sidebar">
+          <ul class="nav nav-sidebar">
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Facilities <span class="caret"></span></a>
+              <ul class="dropdown-menu" role="menu">
+                <li><a href="building.html">Building</a></li>
+                <li class="active"><a href="room.html">Room</a></li>
+              </ul>
+            </li>
+            <li><a href="#">Inventory</a></li>
+            <li><a href="#">Check-In</a></li>
+            <li><a href="#">Reports</a></li>
+          </ul>
+          <ul class="nav nav-sidebar">
+            <li><a href="login.html">Log out</a></li>
+          </ul>
+        </div>
+        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+          <h1 class="page-header">Manage Room</h1>
+          <h4>Building</h4>
+          <div class="bs-docs-example">
+            <input type="text" class="span3" style="margin: 0 auto;" data-provide="typeahead" data-items="4" data-source='["Highland Elementary","Powell County High School"]'>
+          </div>
+          <h4>Room</h4>
+          <div class="bs-docs-example">
+            <input type="text" class="span3" style="margin: 0 auto;" data-provide="typeahead" data-items="4" data-source='["Room 100 - Ms. Smith","Room 101 - Mrs. Jones","Room 102 - Mr. Bill"]'>
+          </div>
+          <br class="divider">
+          <p>
+            <a class="btn btn-primary active" href="#" role="button">Create</a>
+            <a class="btn btn-primary" href="#" role="button">Edit</a>
+            <a class="btn btn-primary" href="#" role="button">Delete</a>
+          </p>
+          <hr class="divider">
+          <form class="form-actions" action"#">
+            <h4>Room Name</h4>
+            <div class="form-group">
+              <input type="text" class="span3" placeholder="Text input" required>
```

Comment:
Avoid generic placeholder text. Maybe describe the sort of data this field wants, e.g., "The room's number and teacher name".